README.md

```diff
@@ -13,6 +13,14 @@ the CRDs:
     kubectl apply -f https://raw.githubusercontent.com/FoundationDB/fdb-kubernetes-operator/master/config/crd/bases/apps.foundationdb.org_foundationdbrestores.yaml
     kubectl apply -f https://raw.githubusercontent.com/foundationdb/fdb-kubernetes-operator/master/config/samples/deployment.yaml
 
+Alternately, use the experimental [helm chart](/charts/fdb-operator/):
```

Comment:
I don't see why we should call the helm chart `experimental`?

Reply:
The whole operator is labeled as experimental, but I see your point. No need to re-iterate if we've already stated it.